docs/docs/developer-guides/cosmwasm/walkthrough.md

```diff
@@ -1,25 +1,25 @@
 # Walkthrough: cw20_base
 
-In this walkthrough, we will upload and interact with `cw20_base` contract on Picasso rococo. We will:
-* Fetch the contract binary from a running Cosmos chain and upload it to our chain.
-* Instantiate the contract.
-* Execute a transfer.
+In this walkthrough, we will upload and interact with a `cw20_base` contract on a local Picasso network by:
+
+* Fetching the contract binary from a running Cosmos chain and upload it to our chain.
+* Instantiating the contract.
+* Executing a transfer.
 
 :::note
-Ensure that you have followed the steps to setup the guide to install clone the `ccw` repository and install nix as outlined in the [first section](https://docs.composable.finance/developer-guides/cosmwasm-cli).
+Ensure that you have followed the guide to setup youre development environment as outlined in the [first section](https://docs.composable.finance/developer-guides/cosmwasm-cli).
 :::
 
-## Running `pallet-cosmwasm` on Picasso rococo
+## Running `pallet-cosmwasm` on Picasso local
+
+**When running Picasso Rococo on a local development network versus on the Picasso Rococo mainnet, the only difference is that you need to replace '-n Alice' with your seed phrase in the commands and change the RPC endpoints.**
```

Comment:
maybe we should add that they need to specify port with rococo rpc endpoint.

Reply:
although it might be clear fro examples

Reply:
do you mean list the RPC endpoints for both there?

Reply:
No, in the command it has port 443 after colon. Up to you to tell them to add the port. Without port it won't work